site/_posts/2021-08-18-annnoucing-fusionauth-1.30.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.30.0
+description: The FusionAuth 1.30.0 releases a robust suite of Advanced Threat Detection features, a JWT vending machine, myriad webhook events, and more.
+author: Akira Brand
+image: blogs/release-1-30/product-update-fusionauth-1.30.png
+category: blog
+tags: topic-troubleshooting
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of version 1.30.0 of FusionAuth. This version shipped on August 12, 2021. 1.30.0 includes a robust Advanced Threat Detection feature, adds 38 webhook events, resolves issues for FusionAuth community members and customers, and more! 
+
+<!--more-->
+
+This release contained a number of features, enhancements, and bug fixes. Please see the [release notes](/docs/v1/tech/release-notes/#version-1-30-0) for a full breakdown of the changes between 1.29 and 1.30.0. 
+
+There are a few items worth calling out.
+
+## JWT Vending Machine
+
+There are scenarios where a user does not yet exist, or is in the process of being created and needs access to services. This may be an anonymous user, or a future user. In these cases we cannot yet authenticate to FusionAuth because the user does not yet exist, or they may not yet have a password.
+
+To solve this, we have created the possiblity of opening an API that allows a JWT to be created with a payload defined by the API called.  This would require an API key, so the caller must be privledged.  
+
+Here is an example of the endpoint at work: 
```

Comment:
I found this example confusing and not really clear. Maybe offer a sample curl script with the input (which you provide) and the output. Or you could omit the curl script and just say something like "when you call the endpoint with the below input: <show input>, you'll get back a signed JWT, which, when decoded and validated, will look like the below output: <show output>"